apps/trading-e2e/src/integration/trading-deal-ticket-order.cy.ts

```diff
@@ -88,5 +88,18 @@ describe('deal ticker order validation', { tags: '@smoke' }, () => {
         'Size cannot be lower than 1'
       );
     });
+    it('must have total margin available', function () {
```

Comment:
totally non-blocking

```suggestion
    it('must have total margin available', () => {
```